config/environments/production.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.configure do
+  config.action_mailer.default_url_options = { host: 'chess-app-knightster.herokuapp.com' }
```

Comment:
I just set up the Heroku app - can you update this to be `knightster.herokuapp.com`?